src/app/settings/organization-plans.component.html

```diff
@@ -234,14 +241,14 @@ <h2 class="spaced-header mb-4">{{ (createOrganization ? 'paymentInformation' : '
             <p class="text-lg"><strong>{{'total' | i18n}}:</strong>
                 {{total | currency:'USD $'}}/{{selectedPlanInterval | i18n}}</p>
         </div>
-        <small class="text-muted font-italic" *ngIf="freeTrial && createOrganization; else paymentChargedImmediately">
+        <!-- <small class="text-muted font-italic" *ngIf="freeTrial && createOrganization; else paymentChargedImmediately">
```

Comment:
Please remove this block (we don't generally keep commented out code)

Reply:
Yep, didn't mean to leave that in there.